src/thorlabs_cube/aqctl_thorlabs_cube.py

```diff
@@ -59,10 +60,12 @@ def main():
                 dev = TpzSim()
             elif product == "kdc101":
                 dev = KdcSim()
+            elif product == "kpz101":
+                dev = KpzSim()
             else:
                 print(
                     "Invalid product string (-P/--product),"
-                    " choose from tdc001, tpz001, or kdc101"
+                    " choose from tdc001, tpz001, or kdc101, kpz101"
```

Comment:
```suggestion
                    " choose from tdc001, tpz001, kdc101, kpz101"
```
For consistency with other message.

Separately - maybe we should set up the choice of device with a different data structure that helps this scale as we add more controllers. Maybe a dictionary that the selection logic and this error message both use?